walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java

```diff
@@ -28,9 +28,9 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Not your fault but could you add a comment that describes what this method does?

Reply:
I actually didn't write this code, it is the code that came when I downloaded all the files given to us! I'm not actually sure what is does.